lib/index.js

```diff
@@ -231,20 +210,13 @@ export class Baker extends EventEmitter {
 
     // if an object of custom nunjucks filters was provided, add them now
     if (nunjucksFilters) {
-      if (crosswalkPath && nunjucksFilters.prepareCrosswalk) {
-        throw new Error(
-          'crosswalkPath and prepareCrosswalk cannot both be defined in nunjucksFilters'
-        );
-      }
-
-      // // Use prepareCrosswalk from nunjucksFilters if available, otherwise check for crosswalkPath
-      if (crosswalkPath && !nunjucksFilters.prepareCrosswalk) {
-        nunjucksFilters.prepareCrosswalk = this.prepareCrosswalk(crosswalkPath);
-      }
-
       this.nunjucks.addCustomFilters(nunjucksFilters);
     }
 
+    if (crosswalkData) {
```

Comment:
After deliberation, I decided that this change will just force people into updating their crosswalk to the new data structure